docs/ONNX_Runtime_for_Mobile_Platforms.md

```diff
@@ -125,46 +124,73 @@ session = onnxruntime.InferenceSession(<path to model>, so)
 ### Enabling Execution Providers that compile kernels in a minimal build
 
 It is possible to enable execution providers that compile kernels in a minimal build. 
-Currently the NNAPI execution provider is the only execution provider that has support for running in a minimal build.
+Currently the NNAPI execution provider is the only compiling execution provider that has support for running in a minimal build.
+
+#### Background
+When ONNX Runtime loads an ONNX format model there are three main steps that affect the operators used in the model, and which execution provider runs each node. 
```

Comment:
I was wondering, is this make the process too complicated or too much details?
Do we want to split this into 2 parts (simple and advanced)

1. SImple,
Use release ORT package (1.6+)
Use basic optimization get ort model
Build Minimal ORT with NNAPI

2. Advanced (TLDR, can be skipped)
Background
Build your own full ORT package
Install the python package
Generate NNAPI aware model
Same Build Steps as the Simple one

Performance caveats ...
...

Reply:
It's a tricky balance. Totally agree there's a lot detail, but if we don't have the detail we'll just have to provide it via Issues created when folks see bad performance. Would it work to create a separate document for advanced usage as you suggested, document the 'simple' approach in the existing doc, and add a link to the advanced doc suggesting that should be considered if perf isn't as expected? perf caveats can go in the advanced doc. 

Hard to know what's best as I have no idea whether having the extended optimizations are likely to be a significant perf gain or not, so not clear how important the advanced usage info will be. 

I also expect this will also get refined a bit based on feedback as it gets tested out.